Makefile

```
@@ -10,6 +10,13 @@ test:
 	poetry run behave
 	poetry run pytest
 
+check-format:
+	poetry run autoflake --check-diff .
+	poetry run black --check --diff .
+	poetry run isort --check .
+	poetry run flake8 .
```

Let's keep the same as in `format` command:
```suggestion
	poetry run flake8 alumnium examples
```
Otherwise, it'll validate the virtual environment folder as well

---

Thanks for suggesting , amended the same